### ISSUE TITLE
Logs are being spammed with Suricata warnings pertaining to duplicate rules

### ISSUE THREAD

**Issue by Njinx** [cisagov/Malcolm#233]
`docker-compose logs` is getting spammed with the following Suricata warning:

```
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230020 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230021 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230027 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230022 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230028 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230023 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230029 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230024 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230030 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230025 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230000 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230026 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230001 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230027 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230015 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230002 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230028 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230003 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230029 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230004 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230030 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230007 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230000 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230009 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230001 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230010 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230015 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230011 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230002 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230012 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230003 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230013 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230004 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230014 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230007 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230016 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230009 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230017 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230010 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230018 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230011 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230019 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230012 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230020 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230013 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230021 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230014 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230022 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230016 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230017 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230023 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230018 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230024 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230019 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230025 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230020 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230026 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230021 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230027 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230022 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230028 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230023 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230029 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:03Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230030 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230024 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:31Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230028 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230025 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:31Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230029 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230026 with same revision, keeping the first rule seen.
suricata-live_1       | 2023-01-30T18:00:31Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:03 - <Warning> -- Found duplicate rule SID 2230030 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230027 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230027 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230028 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230029 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:04Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230030 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:30Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230028 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:30Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230029 with same revision, keeping the first rule seen.
suricata_1            | 2023-01-30T18:00:30Z  /bin/bash /usr/local/bin/suricata-update-rules.sh: 30/1/2023 -- 18:00:04 - <Warning> -- Found duplicate rule SID 2230030 with same revision, keeping the first rule seen.
```

These go on for as far as the eye can see. I know very little about Suricata and certainly didn't create any duplicate rules. 

I'm running a mostly vanilla instance that's just been gathering traffic from a TAP as I learn how Malcolm works. I'm happy to provide more information on request.

**Comment by mmguero** [comment]
I'll look into it. After the periodic rule update finishes I would assume that this ends, at least until the next potential update. The warning might be something I just end up filtering out from the logs.

**Comment by Njinx** [comment]
I believe you're right. It's been a couple hours and the log messages are gone. I don't know anything about the rule update process (yet) but would it be possible to fix this issue without suppressing _all_ duplicate rule warnings? They could be of help if you were to actually have duplicate rules.

**Comment by mmguero** [comment]
Yeah, my guess is what's happening is I've got suricata's "default" ruleset built into Malcolm, and then the update process is grabbing updates to those same rules. I think you're exactly right, I'll just need to supress those duplicate warning messages.

**Comment by mmguero** [comment]
I'll just filter out the ones that have the same revision, duplicate rules that are actually updates or that otherwise have different revisions will still put out the warning.

**Comment by mmguero** [comment]
Fixed via mmguero-dev/Malcolm@1736286
